qiskit-ibm-runtime-mcp-server/README.md

```diff
@@ -26,20 +26,47 @@ This project recommends using [uv](https://astral.sh/uv) for virtual environment
    uv sync
    ```
 
-2. **Configure environment variables**:
+2. **Get your IBM Quantum token** (if you don't have saved credentials):
+   - Visit [IBM Quantum](https://quantum.cloud.ibm.com/)
+   - Sign up or log in to your account
+   - Go to Account Settings
+   - Copy your API token
```

Comment:
not sure if i am missing something here but these steps do not work for me. i do not see my API token or option to copy it in the account settings page